syft/workers/base.py

```diff
@@ -691,7 +691,10 @@ def __repr__(self):
         return self.__str__()
 
     def __getitem__(self, idx):
-        return self._objects[idx]
+        try:
```

Comment:
This is equivalent to self._objects.get(idx,None) which looks a lot cleaner

Reply:
or the default default is None so just .get(idx)

Reply:
the default is to throw a KeyError I believe :)

Reply:
fixed